GVRf/Framework/jni/contrib/assimp/include/assimp/scene.h

```diff
@@ -264,7 +267,13 @@ struct aiNode
  */
 #define AI_SCENE_FLAGS_TERRAIN 0x10
 
+enum aiTextureWrapAxis

```

Comment:
Wrap parameters exist in include/assimp/material.h. Please check if those values are already parsed correctly from FBX file. If so, you can pass those parameters to GVRTexture. Generally, it is probably problematic to set wrapping parameters globally (for the whole scene).